airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -42,7 +42,10 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
     final boolean useAsyncSnowflake = false;
     // final boolean useAsyncSnowflake = config.has("loading_method")
     // && config.get("loading_method").has("method")
-    // && config.get("loading_method").get("method").asText().equals("Internal Staging");
+    // && config.get("loading_method").get("method").asText().equals("Internal Staging")
+    // // Standard is when someone doesn't specify a loading method but still
+    // // DestinationType.INTERNAL_STAGING
+    // || config.get("loading_method").get("method").asText().equals("Standard");
```

Comment:
`Standard` is the config option when someone doesn't explicitly set `Internal Staging`. This is within the `spec.json` under the `destination-snowflake` connector folder

Reply:
What about `"INTERNAL_STAGING"`?

Reply:
I tested this locally and am actually not sure where `INTERNAL_STAGING` came from. I was successfully able to get an async version running when not setting the value and matching the loading method to `Standard`

https://gist.github.com/ryankfu/b5f2089ac722ae7e4c7592bcb942b2f0. Given that, I don't see a need to add `INTERNAL_STAGING` especially since the plan is simply have `isAync` = true meaning all loading methods regardless of `loading_method` will be on. Not strongly opinionated here